apps/cli/src/sky/writeConfigToFiles.ts

```diff
@@ -57,8 +60,19 @@ export const writeConfigToFiles = async (opts: {
               createTypeGenFile: runTypeGen ? writeToFiles : undefined,
             });
 
-            await modifySkyConfigSource({ filePath: opts.uri });
-            console.log(`${opts.uri} - updated with sky config`);
+            const fileContents = await fs.readFile(opts.uri, 'utf8');
+            const code = await modifySkyConfigSource({
+              fileContents,
+              filePath: opts.uri,
+            });
+            if (code) {
+              const formattedCode = await getPrettierInstance(opts.cwd).format(
+                code,
+                { parser: 'typescript' },
```

Comment:
in the other place we are doing smth like this
```suggestion
                {
                  ...(await prettierInstance.resolveConfig(opts.uri)),
                  parser: 'typescript'
                },
```

Reply:
updated in latest commit